tools/validators/instance_validator/tests/fake_instances/BAD/translation_invalid_udmi_present_value_name.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PHYSICAL-ENTITY-GUID:
+  type: GATEWAYS/PASSTHROUGH
+  code: PHYSICAL-ENTITY
+  cloud_device_id: "2619178366980754"
+  translation:
+    differential_pressure_specification:
+      # Missing `present_value` suffix.
+      present_value: "points.differential_pressure_specification"
+      units:
+        key: "points.differential_pressure_specification.present_value.units"
+        values:
+          pascals: "pascals"
+
+PHYSICAL-ENTITY-GUID-2:
+  type: GATEWAYS/PASSTHROUGH
+  code: PHYSICAL-ENTITY
+  cloud_device_id: "2619178366980755"
+  translation:
+    differential_pressure_specification:
+      # Only the field name can exist between `points` and `present_value`.
+      present_value: "points.differential_pressure_specification.garbage.present_value"
+      units:
+        key: "pointset.points.differential_pressure_specification.units"
+        values:
+          pascals: "pascals"
+
+PHYSICAL-ENTITY-GUID-3:
+  type: GATEWAYS/PASSTHROUGH
+  code: PHYSICAL-ENTITY
+  cloud_device_id: "2619178366980756"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.